### DISCUSSION TITLE
border bug

### DISCUSSION THREAD

**Discussion by lolkeksanctionblocked** [tauri-apps/tauri#11980] [Q&A]
```
"windows": [
      {
        "title": "Ульта",
        "width": 282,
        "height": 256,
        "resizable": false,
        "fullscreen": false,
        "decorations": false,
        "transparent": true
        
      }
    ],
```

Help pls, got this strange border on win10. Win 11 works fine. Tried disable shadow, but this solution on windows 11 sucks.  
![image](https://github.com/user-attachments/assets/0a335d46-ea59-4687-b88b-758953e254a5)

**Comment by FirstSlip** [comment]
Same problem

**Comment by FabianLars** [comment]
This is tracked here https://github.com/tauri-apps/tauri/issues/11654 - i'll also close this discussion as a duplicate of that one.

Also, first and last warning, if you ever have the audacity again to look me up on another fucking messenger platform just to make me look at your issue (that you apparently tried to solve for a week without ever seeing #11654?) i will ban you from interacting with this github org.
